src/modules/Schema/containers/CreateSchema/index.types.ts

```diff
@@ -0,0 +1,16 @@
+export type AttributeOption = {
```

Comment:
please update everything with main and rebase it (because I changed these files in `Feat/API-Schema`)
please rebase and drop my commits from your PR and get the updated main into your PR